### ISSUE TITLE
Mousing over mob increments red item looted counter

### ISSUE THREAD

**Issue by fireundubh** [TheRealHypo/MobInfo2#13]
Whenever I mouse over an affected mob, the counter for `2725` is incremented.

![Imgur](https://i.imgur.com/SMqi1c0.jpg)

`2725` is the item ID of `Green Hills of Stranglethorn - Page 1`, but still, the counter shouldn't increment on click.

**Comment by fireundubh** [comment]
The counter only increments if "combine same mobs" is disabled.

**Comment by ghost** [comment]
Yeah, since Item Values are only shown if combine sae mobs is disabled... I don't know what Item Value this is 🤔  

Might need some help on this, or I'll have to check out Stranglethorn when I get lvl 30+ finally :D
